web-api/src/config/load_config.rs

```diff
@@ -78,24 +78,30 @@ where
         figment = figment.join(Env::prefixed(&format!("{name}__")).split("__"));
     }
 
-    let provider = config
-        .map(|content_or_path| {
-            if let Some(content) = content_or_path.strip_prefix("inline:") {
-                Toml::string(content)
+    let provider = if let Some(content_or_path) = config {
+        if let Some(content) = content_or_path.strip_prefix("inline:") {
+            Toml::string(content)
+        } else {
+            let path = Path::new(content_or_path);
+            if path.is_file() {
+                Toml::file(path)
             } else {
-                Toml::file(content_or_path)
+                return Err(figment::Error::from(format!(
+                    "Config file missing or not a file: {}",
+                    path.display()
```

Comment:
:art: trailing comma
```suggestion
                    path.display(),
```